userspace/falco/stats_writer.cpp

```diff
@@ -84,6 +84,12 @@ stats_writer::stats_writer(
 	: m_initialized(false), m_total_samples(0)
 {
 	m_config = config;
+	// capacity and controls should not be relevant for stats outputs, adopt capacity
+	// for completeness, but do not implement config recovery strategies.
+	if (config->m_outputs_queue_capacity > 0)
```

Comment:
same as before if we use `-1` as default

Reply:
will do